.travis.yml

```diff
@@ -9,7 +9,6 @@ env:
     - PACKAGE=fiat_p256
     - DISTRO=alpine-3.7
   matrix:
-    - OCAML_VERSION=4.04
```

Comment:
`ocamlformat 0.8` is not compatible with `ocaml < 4.05`.

Reply:
sounds good, can you add `"ocaml" >= "4.05.0"` to the opam file? (or whatever is the opam 2 way to set a constraint on the compiler)